pkg/vcs/testos.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package vcs
+
+import (
+	"io"
+)
+
+type Testos struct {
+	*git
+}
+
+func NewTestos(dir string) *Testos {
```

Comment:
If you add "test" OS to vcs.NewRepo as you did for pkg/build, then we can remove custom creation of vcs instance from pkg/bisect, it will just call vcs.NewRepo for the "test" OS and get what we want. It looks reasonable to do.